### DISCUSSION TITLE
Rename your home Wi-Fi network

### DISCUSSION THREAD

**Discussion by bceda** [security-mountain/security-mountain.github.io#7] [Ideas]
Is obscurity security?

Renaming your home network might make your network appear to not run a specific make/model of router, but aren't their other ways to find this information out?

**Comment by apollodelphi702** [comment]
On the flip side - I would argue making that information harder to come by is not a bad thing and this is an easy thing to do to avoid giving the hackers very easy access to the information about the make/model of your router.

Then again, perhaps it's worth adding guidance on _what to rename your router to_ - certain names (e.g., <your name>'s Wi-Fi Network) probably divulge _more_ information than the original router name.
